src/Integration.Vsix/Manifests/VS2022/source.extension.vsixmanifest

```diff
@@ -34,6 +34,8 @@
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="File" Path="SonarAnalyzer.CSharp.dll" />
         <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="File" Path="SonarAnalyzer.VisualBasic.dll" />
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="File" Path="SonarAnalyzer.VisualBasic.dll" />
+        <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="File" Path="SonarAnalyzer.CFG.dll" />
+        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="File" Path="SonarAnalyzer.CFG.dll" />
```

Comment:
Adding these settings to the manifest was enough to make the analyzers work on @georgii-borovinskikh-sonarsource's machine.

As expected, we then saw the same failure for our `Roslyn.Suppressions` assembly. I'll fix that separately.

We only tested on VS2022, but I've made the same changes in the VS2019 manifest.